ccd-definition/ComplexTypes/CaseManagementOrder.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "CaseManagementOrder",
+    "ListElementCode": "hearingDate",
+    "FieldType": "Text",
+    "ElementLabel": "The date of the hearing",
```

Comment:
as discucsed please put XXX

Reply:
Done this change

Reply:
@Moray suggest that the text will be the same as question label "Which hearing is this order for?". It is the same pattern across.